### DISCUSSION TITLE
issue in setting up the project

### DISCUSSION THREAD

**Discussion by Harshul02** [Stoccoin-Official/Stoccoin-Website#223] [General]
While setting up the project, it is showing

Failed to resolve import "@fortawesome/react-fontawesome" from "src\components\Footer.jsx". Does the file exist?

I tried even installing this module again, but not able to resolve this error and set up the project. Please help.

**Comment by AbhayKrSing** [comment]
Your issue get resolved??

**Reply by Harshul02** [reply]
yes
